### PR TITLE
Import and adapt Compound Governor

### PR DESCRIPTION
**Pull request by cloudhead**
This is the first of several changes to address #71. Here, we just import the compound governor and token, and adapt them so that they compile.

I've conserved the original contract, to keep track of the changes I've made (c57577d).

You can find the originals here: https://github.com/compound-finance/compound-protocol/tree/master/contracts/Governance

### CODE REVIEW

File: contracts/Governance/SafeMath.sol [hunk 1]
```diff
@@ -0,0 +1,185 @@
+pragma solidity ^0.7.5;
```
Comment (CodeSandwich) [inline contracts/Governance/SafeMath.sol:1]: Don't we already have `contracts/libraries/SafeMath.sol`? :thinking:

Comment (cloudhead) [reply contracts/Governance/SafeMath.sol:1]: Hah, good catch.

File: contracts/Governance/Token.sol [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+pragma solidity ^0.7.5;
+pragma experimental ABIEncoderV2;
+
+contract Token {
```
Comment (geigerzaehler) [inline contracts/Governance/Token.sol:30]: Could we prefix this with `Governance`? Otherwise it shows up as generic `Token` everywhere else.

Comment (cloudhead) [reply contracts/Governance/Token.sol:30]: I will rename to `RadicleToken`. Then, I will remove `Rad.sol` and point everything to `RadicleToken.sol`.

**Review by CodeSandwich — APPROVED** [review]
LGTM!

**Pull request merged**